packages/pn-personagiuridica-webapp/src/component/Deleghe/DelegatesByCompany.tsx

```diff
@@ -3,7 +3,18 @@ import { useNavigate } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import { Box, Button, Chip, Stack, Typography } from '@mui/material';
 import AddIcon from '@mui/icons-material/Add';
-import { Column, ItemsTable as Table, Item, CodeModal, Sort, EmptyState, ApiErrorWrapper } from '@pagopa-pn/pn-commons';
+import {
```

Comment:
I should remove all the code that is useless for the purpose of the issue

Reply:
done